pkg/vault/test/authorizer/TimelockAuthorizerExecute.test.ts

```diff
@@ -253,6 +253,12 @@ describe('TimelockAuthorizer execute', () => {
       const where = await authorizer.instance.getTimelockExecutionHelper();
       await expect(schedule(where)).to.be.revertedWith('ATTEMPTING_EXECUTION_HELPER_REENTRANCY');
     });
+
+    it('reverts if data is less than 4 bytes', async () => {
+      await expect(authorizer.schedule(authenticatedContract.address, '0x00', [], { from: user })).to.be.revertedWith(
+        'SENDER_DOES_NOT_HAVE_PERMISSION'
```

Comment:
Hm not a fan of this error message. Let's change `_decodeSelector` so that it reverts if it cannot get a selector.

Reply:
Created https://github.com/balancer/balancer-v2-monorepo/issues/2380